### ISSUE TITLE
tarball data for react-timeago-i18n-2.2.0 seems to be corrupted

### ISSUE THREAD

**Issue by mmrein** [k-yle/react-timeago-i18n#4]
Hi, just wanted to update to v2.2.0 and got sha512 mismatch from npm registry, here is full log:

```
167 silly tarball no local data for react-timeago-i18n@https://registry.npmjs.org/react-timeago-i18n/-/react-timeago-i18n-2.2.0.tgz. Extracting by manifest.
168 http fetch POST 200 https://registry.npmjs.org/-/npm/v1/security/advisories/bulk 392ms
169 timing auditReport:getReport Completed in 408ms
170 silly audit report {}
171 timing auditReport:init Completed in 0ms
172 timing reify:audit Completed in 410ms
173 warn tarball tarball data for react-timeago-i18n@https://registry.npmjs.org/react-timeago-i18n/-/react-timeago-i18n-2.2.0.tgz (sha512-UxL+4whBp5EoT0n3qyMBObVgiAyoOfDjgagycDTEvGnJ0xHrfmqQfg391K+ibYazsukKZ9I7n4H4YuH4WcF1fw==) seems to be corrupted. Trying again.
174 http fetch GET 200 https://registry.npmjs.org/react-timeago-i18n/-/react-timeago-i18n-2.2.0.tgz 293ms (cache miss)
175 warn tarball tarball data for react-timeago-i18n@https://registry.npmjs.org/react-timeago-i18n/-/react-timeago-i18n-2.2.0.tgz (sha512-UxL+4whBp5EoT0n3qyMBObVgiAyoOfDjgagycDTEvGnJ0xHrfmqQfg391K+ibYazsukKZ9I7n4H4YuH4WcF1fw==) seems to be corrupted. Trying again.
176 http fetch GET 200 https://registry.npmjs.org/react-timeago-i18n/-/react-timeago-i18n-2.2.0.tgz 91ms (cache miss)
177 timing reify:rollback:createSparse Completed in 7ms
178 timing reify:rollback:retireShallow Completed in 0ms
179 timing command:install Completed in 8774ms
180 verbose stack Error: sha512-UxL+4whBp5EoT0n3qyMBObVgiAyoOfDjgagycDTEvGnJ0xHrfmqQfg391K+ibYazsukKZ9I7n4H4YuH4WcF1fw== integrity checksum failed when using sha512: wanted sha512-UxL+4whBp5EoT0n3qyMBObVgiAyoOfDjgagycDTEvGnJ0xHrfmqQfg391K+ibYazsukKZ9I7n4H4YuH4WcF1fw== but got sha512-BQ9hrvg8heZh1xlGsyzuHOfbK3sFnenTwqwBAkGY5jdAo32Zbe60ZEcs/VGwrHnungiS5UAhJlF92d/xd7/FJQ==. (5641 bytes)
180 verbose stack     at #onEnd (/usr/lib/node_modules/npm/node_modules/ssri/lib/index.js:111:19)
180 verbose stack     at IntegrityStream.emit (/usr/lib/node_modules/npm/node_modules/ssri/lib/index.js:81:18)
180 verbose stack     at [maybeEmitEnd] (/usr/lib/node_modules/npm/node_modules/minipass/dist/commonjs/index.js:713:18)
180 verbose stack     at IntegrityStream.emit (/usr/lib/node_modules/npm/node_modules/minipass/dist/commonjs/index.js:794:29)
180 verbose stack     at IntegrityStream.emit (/usr/lib/node_modules/npm/node_modules/ssri/lib/index.js:83:18)
180 verbose stack     at [resume] (/usr/lib/node_modules/npm/node_modules/minipass/dist/commonjs/index.js:484:18)
180 verbose stack     at IntegrityStream.resume (/usr/lib/node_modules/npm/node_modules/minipass/dist/commonjs/index.js:496:28)
180 verbose stack     at CachingMinipassPipeline.emit (/usr/lib/node_modules/npm/node_modules/minipass-pipeline/index.js:105:19)
180 verbose stack     at CachingMinipassPipeline.emit (/usr/lib/node_modules/npm/node_modules/make-fetch-happen/lib/pipeline.js:37:18)
180 verbose stack     at [resume] (/usr/lib/node_modules/npm/node_modules/minipass-pipeline/node_modules/minipass/index.js:312:10)
181 verbose cwd /mnt/d/REDACTED
182 verbose Linux 4.4.0-19041-Microsoft
183 verbose node v20.11.1
184 verbose npm  v10.2.4
185 error code EINTEGRITY
186 error sha512-UxL+4whBp5EoT0n3qyMBObVgiAyoOfDjgagycDTEvGnJ0xHrfmqQfg391K+ibYazsukKZ9I7n4H4YuH4WcF1fw== integrity checksum failed when using sha512: wanted sha512-UxL+4whBp5EoT0n3qyMBObVgiAyoOfDjgagycDTEvGnJ0xHrfmqQfg391K+ibYazsukKZ9I7n4H4YuH4WcF1fw== but got sha512-BQ9hrvg8heZh1xlGsyzuHOfbK3sFnenTwqwBAkGY5jdAo32Zbe60ZEcs/VGwrHnungiS5UAhJlF92d/xd7/FJQ==. (5641 bytes)
187 verbose exit 1
188 timing npm Completed in 9924ms
189 verbose unfinished npm timer reify 1718792469533
190 verbose unfinished npm timer reify:unpack 1718792477979
191 verbose unfinished npm timer reifyNode:node_modules/react-timeago-i18n 1718792477983
192 verbose code 1
```

Not sure if this could be something on my end?

**Comment by k-yle** [comment]
hey, thanks for opening an issue about this, it seems like it was caused by a race condition in the CI. Ironically, [the npm website](https://npm.im/react-timeago-i18n) says that the release was correctly signed with [npm provenance](https://github.blog/2023-04-19-introducing-npm-package-provenance) despite the shasums not matching...

The CI has just compiled & published `v2.2.1`, which seems to work now

**Comment by mmrein** [comment]
v2.2.1 updated without issues, `allowFuture` is now working.

Awesome, thanks for quick response!
